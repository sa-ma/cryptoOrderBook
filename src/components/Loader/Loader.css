.loader {
  margin: 20vh auto;
  width: 50%;
  text-align: center;
}
