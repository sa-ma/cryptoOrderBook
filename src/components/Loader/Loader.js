import React from 'react';
import loadingGif from '../../assets/loading.svg';
import './Loader.css';

const Loader = () => {
  return (
    <div className="loader">
      <img src={loadingGif} alt="loading animation" />
    </div>
  );
};

export default Loader;
