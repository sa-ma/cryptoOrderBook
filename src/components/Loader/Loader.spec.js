import React from 'react';
import { shallow } from 'enzyme';
import Loader from './Loader';

describe('<Loader/> Component', () => {
  it('should render app component without crashing', () => {
    const component = shallow(<Loader />);
    expect(component).toMatchSnapshot();
  });
});
