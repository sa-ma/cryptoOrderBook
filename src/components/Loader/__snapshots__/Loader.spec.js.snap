// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader/> Component should render app component without crashing 1`] = `ShallowWrapper {}`;
